backend/src/modules/task/controllers/task.controller.ts

```diff
@@ -15,7 +28,28 @@ export class TaskController {
 
   @Post()
   async create(@Req() req: AuthenticatedRequest, @Body() dto: CreateTaskDto) {
-    const user = req.user; 
+    const user = req.user;
     return await this.taskService.createTask({ ...dto, createdBy: user.id });
   }
-}
+
+  @Get()
+  async getAll() {
+    return await this.taskService.getAllTasks();
+  }
+
+  @Patch(':id')
+  async editTask(
+    @Param('id', ParseIntPipe) id: number,
+    @Body() dto: EditTaskDto
+  ) {
+    return await this.taskService.editTask(id, dto);
+  }
```

Comment:
please make sure the PR has only related changes, since this is delete related task, I expect to see only changes for delete